gtsfm/configs/deep_front_end_astronet.yaml

```diff
@@ -0,0 +1,68 @@
+# Relaxes triangulation RANSAC threshold to 100 (10x threshold of deep_front_end.yaml)
```

Comment:
Can we name it something which mentions this attribute value being high instead of tagging it with a database?

Reply:
I agree, I think this would improve clarity

Reply:
AstroNet is the only dataset that uses the config by default, so it seems fitting to name it accordingly.